school_center.py

```diff
@@ -9,7 +9,10 @@
 import random
 import argparse
 from typing import Dict, List
+import logging
+import sys
 
+logging.basicConfig(level=logging.ERROR)
```

Comment:
I think we can just use the logger as follows. No need to use the logging module and basiconfig, we can set the default log level if needed in the future.

```python
from logging import getLogger 
logger = getLogger(__name__) `
logger.info/error("Lovely message")